javascript/README.md

```diff
@@ -69,9 +69,7 @@ project:
 
 4. Run the Closure Compiler to get your changes syntax and type checked. This will also generate `demo-compiled.js` used by `demo-compiler.html`
 
-  (TODO: Use target "compile" rather than "compile-demo" once visibility issues for getExtractedNationalPrefix_ have been resolved):
```

Comment:
Indeed, it works, even though the visibility issues haven't been resolved. If it stops working, I also know how to fix it - by moving the flags that care about visibility into the demo rule only, not the test rule.